client/src/amplify-ui/sidebar-layout/sidebar-layout-toggle/sidebar-layout-toggle.tsx

```diff
@@ -20,10 +20,10 @@ export class AmplifySidebarLayoutToggle {
    * sidebar (mobile only)
    */
   getClass = () => {
-    return {
-      [sidebarLayoutSidebarToggleStyle]: true,
-      ...(this.inViewClass ? {[this.inViewClass]: !!this.inView} : {}),
-    };
+    return (
+      sidebarLayoutSidebarToggleStyle +
+      (this.inViewClass && this.inView ? " " + this.inViewClass : "")
+    );
```

Comment:
This change probably didn't need to occur, but it looks a lot cleaner this way to my eyes.